src/me/mrCookieSlime/Slimefun/Setup/Messages.java

```diff
@@ -30,6 +30,7 @@ public static void setup() {
 		local.setDefault("messages.fortune-cookie", "&7Help me, I am trapped in a Fortune Cookie Factory!", "&7You will die tomorrow...     by a Creeper", "&7At some Point in your Life something bad will happen!!!", "&7Next week you will notice that this is not the real World, you are in a kind of \"Matrix\" or lets call it Computer game. Yes you are in a Computer Game", "&7This Cookie will taste good in a few seconds", "&7You will die soon and the last word you will hear is gonna be \"EXTERMINATE!!!\"", "&7Whatever you do, do not hug a Creeper... I tried it. It feels good, but it's not worth it.");
 		local.setDefault("messages.only-players", "&4This Command is only for Players");
 		local.setDefault("messages.no-permission", "&4You do not have the required Permission to do this");
+		local.setDefault("messages.item-permission", "&rYou do not have Permission", "&rto access this Item");
```

Comment:
This is not a message and therefore shouldn't go into that namespace.
I'd suggest the prefix "tooltips." instead.

Reply:
Done